<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>IrisAnalysisUI</class>
 <widget class="QMainWindow" name="IrisAnalysisUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>786</width>
    <height>764</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="imageView">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>115</y>
      <width>596</width>
      <height>371</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="imageFileName">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>491</width>
      <height>36</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="lineWidth">
     <number>1</number>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>10</y>
      <width>731</width>
      <height>41</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="loadImageBtn">
       <property name="text">
        <string>Load Image</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="zoomInBtn">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>+</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="zoomOutBtn">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>-</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="scale1Btn">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>1</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="analysis0Btn">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Anaysis0</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="analysis1Btn">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Analysis 1</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="qualityValue">
       <property name="autoFillBackground">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>Quality:0</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>786</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <tabstops>
  <tabstop>loadImageBtn</tabstop>
  <tabstop>zoomInBtn</tabstop>
  <tabstop>zoomOutBtn</tabstop>
  <tabstop>scale1Btn</tabstop>
  <tabstop>analysis0Btn</tabstop>
  <tabstop>analysis1Btn</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
